src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/list/AdminAreasOfLawEndpointTest.java

```diff
@@ -222,8 +308,47 @@ private AreaOfLaw getCurrentAreaOfLaw() {
 
     private AreaOfLaw updateAreaOfLaw() {
 
-        return new AreaOfLaw(AREA_OF_LAW_ID_HOUSING_POSSESSION,TEST_NAME,TEST_SINGLE_POINT_OF_ENTRY,TEST_EXTERNAL_LINK,
-                                                          TEST_EXTERNAL_LINK_DESC,TEST_EXTERNAL_LINK_DESC_CY,TEST_ALT_NAME,TEST_ALT_NAME_CY,TEST_DISPLAY_NAME,
-                                                          TEST_DISPLAY_NAME_CY,TEST_DISPLAY_EXTERNAL_LINK);
+        return new AreaOfLaw(
```

Comment:
The name of the method is `updateAreaOfLaw` but it is not doing so. Is it not returning a new AreaOfLaw? Might need a name change on the method. A small one